lib/portlayer/portlayer.go

```diff
@@ -103,6 +103,20 @@ func Init(ctx context.Context, sess *session.Session) error {
 	return nil
 }
 
+func Finalize(ctx context.Context) error {
+	op := trace.NewOperation(context.Background(), "Shutdown")
+	defer trace.End(trace.Begin("", op))
+
+	storage.Finalize(op)
+	// store.Finalize(ctx)
+	exec.Finalize(op)
+	// network.Finalize(ctx)
+	// logging.Finalize(ctx)
```

Comment:
Why are some of these Finalize calls commented out? I don't see an explanation here, in the commit message, or in the PR summary.

Reply:
Because at the moment the methods don't exist as there's no Finalization required for session handling in those packages.
I'll add the stub functions in.